### ISSUE TITLE
Local News Section in services

### ISSUE THREAD

**Issue by matteomessmer** [matteomessmer/hacktoberfest#25]
Here we will discuss about the local news service.
I think one way to get the user location could be based on its public ip

Issue assigned to @Sawsanalshaghel

**Comment by Sawsanalshaghel** [comment]
Hi
could you please explain why you need the user location?
Isn't the issue to build a local news website?

**Comment by Sawsanalshaghel** [comment]
Hi
could you please explain why you need the user location?
Isn't the issue to build a local news website?

**Comment by matteomessmer** [comment]
Yes, we could use the location to retrieve only the news in that particular place, it was just an idea, we can do it in other ways
